README.md

```diff
@@ -97,37 +97,10 @@ This signal is sent when a translation from RWS LanguageCloud is successfully im
 - **source_object:** The page or snippet instance that this translation is for.
 - **translated_object:** The translated instance of the page or snippet.
 
-Here’s how you could use it to send Slack notifications.
-
-```python
-from wagtail_localize.models import get_edit_url
-from wagtail_localize_rws_languagecloud.signals import translation_imported
-import requests
-
-# Let everyone know when translations come back from RWS LanguageCloud
-def send_to_slack(sender, instance, source_object, translated_object, **kwargs):
-    url = (
-        "https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX"
-    )
-
-    edit_url = "https://www.mysite.com" + get_edit_url(translated_object)
-
-    values = {
-        "text": f"Translated content for '{translated_object}' is ready for review at: {edit_url}",
-        "channel": "#translation-notifications",
-        "username": "Wagtail",
-        "icon_emoji": ":rocket:",
-    }
-
-    requests.post(url, values)
-
-
-# Register a receiver
-translation_imported.connect(send_to_slack)
-```
-
 For more information on signal handlers, see [the Django docs](https://docs.djangoproject.com/en/stable/topics/signals/#connecting-receiver-functions).
 
+There are some more detailed examples of how the `translation_imported` signal can be used to send slack notifications in [slack-notification-examples.md](docs/slack-notification-examples.md).
```

Comment:
I decided to move all this out to a separate file because otherwise about half the README would be taken up with just slightly different examples about how to use this one feature and I think it is good to keep the README a fairly concise overview. Tbh, this package has probably now reached the size where it warrants a small sphinx/mkdocs site to go with it but clearly we are not going to shave that yak right now :cow2:  (although I would quite like to).